helpline-dialer.php

```diff
@@ -8,13 +8,20 @@
     $phone_number = getHelplineVolunteer($service_body_id, $tracker);
     $call_status = isset($_REQUEST["DialCallStatus"]) ? $_REQUEST["DialCallStatus"] : "starting";
     $call_timeout = isset($GLOBALS["call_timeout"]) ? $GLOBALS["call_timeout"] : 20;
+
+    if (isset($GLOBALS['forced_callerid']) && $GLOBALS['forced_callerid']) { 
+      $caller_id = $GLOBALS["forced_callerid"];
+    } else { 
+      $caller_id = isset($_REQUEST["Called"]) ? $_REQUEST["Called"] : "0000000000";
+    } 
```

Comment:
This block should've still been moved down.  Going to fix it.

Reply:
ok